lib/bap_plugins/bap_plugins.mli

```diff
@@ -85,30 +85,33 @@ module Std : sig
       | `Errored of string * Error.t (** failed to load a plugin    *)
     ] [@@deriving sexp_of]
 
-    (** [list ?library ()] scans a list of directories, provided by a
+    (** [list ?query ?library ()] scans a list of directories, provided by a
```

Comment:
`query` is a bad name for the parameter. The better names would be `provides`, The idea is to look at the point of invocation: `list ~provides:["disassembler"; "x86"]` is much better than `list ~query:["disassembler"; "x86"]`.